pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/com/pubnub/api/PubNubImpl.kt

```diff
@@ -354,11 +354,14 @@ class PubNubImpl(val jsPubNub: PubNubJs) : PubNub {
         channels: List<String>,
         channelGroups: List<String>,
         includeState: Boolean,
-        includeUUIDs: Boolean
+        includeUUIDs: Boolean,
+        limit: Int,
```

Comment:
Isn't limit optional, and if missing, assigned to default to enforce maximum value?
If this function is not called directly by the user, then it shouldn't be a problem.

Reply:
In kotlin/pubnub-kotlin/pubnub-kotlin-api/src/commonMain/kotlin/com/pubnub/api/PubNub.kt there is proper definition:

fun hereNow(
    channels: List<String> = emptyList(),
    channelGroups: List<String> = emptyList(),
    includeState: Boolean = false,
    includeUUIDs: Boolean = true,
    limit: Int = 1000,
    offset: Int? = null,
): HereNow